master/pkg/model/experiment_config.go

```diff
@@ -88,7 +88,6 @@ type ResourcesConfig struct {
 	Weight         float64      `json:"weight"`
 	NativeParallel bool         `json:"native_parallel,omitempty"`
 	ShmSize        *StorageSize `json:"shm_size,omitempty"`
-	AgentLabel     string       `json:"agent_label"`
```

Comment:
I think a NTSC will error with an agent label specified but an experiment with a label specified will just be ignored? This difference seems somewhat not ideal

Reply:
great catch.